main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -2387,6 +2389,20 @@ object Parser2 {
       close(mark, TreeKind.Expr.Do)
     }
 
+    private def invokeMethod2Expr()(implicit s: State): Mark.Closed = {
```

Comment:
Can you give an example of the current syntax we allow?

is it `name#method(exps).method(exps)`?

Reply:
Sorry, I should have added more comments.
The followings are valid expressions, example gave by @herluf-ba:
`def foo(myObj: String): String \ IO =
    let a = myObj#method();
    let b = MyObj#method();
    let c = myObj#method(x, 1+5);
    myObj#toUpperCase()`